src/app/Library/CrudPanel/Traits/Update.php

```diff
@@ -113,10 +113,6 @@ private function getModelAttributeValueFromRelationship($model, $field)
                 if (! isset($field['subfields'])) {
                     return $related_model->{$relation_method};
                 }
-                // we want to exclude the "self pivot field" since we already have it.
```

Comment:
We added `->getAttributes()` for the model in a previous PR, so we don't rely anymore on the pivot fields to get the values from database, we just get the pivot values.